nym-vpn-lib/src/routing.rs

```diff
@@ -233,21 +236,15 @@ pub async fn setup_routing(
         device_mtu = dev.as_ref().mtu(),
     );
 
+    let _ipv6_addr = config.tun_ips.ipv6.to_string();
```

Comment:
nitpick: remove leading _

Reply:
This is to not have warnings on building for platforms different than linux and macos